services/web/server/src/simcore_service_webserver/workspaces/_exceptions_handlers.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from servicelib.aiohttp import status
+
+from ..exceptions_handlers import (
+    ExceptionToHttpErrorMap,
+    HttpErrorInfo,
+    create_exception_handlers_decorator,
+)
+from ..projects.exceptions import (
+    BaseProjectError,
+    ProjectRunningConflictError,
+    ProjectStoppingError,
+)
+from .errors import (
+    WorkspaceAccessForbiddenError,
+    WorkspaceGroupNotFoundError,
+    WorkspaceNotFoundError,
+    WorkspacesValueError,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+_TO_HTTP_ERROR_MAP: ExceptionToHttpErrorMap = {
+    WorkspaceGroupNotFoundError: HttpErrorInfo(
+        status.HTTP_404_NOT_FOUND,
+        "Workspace {workspace_id} group {group_id} not found.",
+    ),
+    WorkspaceAccessForbiddenError: HttpErrorInfo(
+        status.HTTP_403_FORBIDDEN,
+        "Does not have access to this workspace",
```

Comment:
is there not something missing in this error? like {user_id} or something similar?

Reply:
We can do that. This here is the error sent to the user. It is a template as well and can use the context capture in the exception.